<?xml version="1.0" encoding="ISO-8859-15"?>

<launch>
    <arg name="machine"/>
    <rosparam file="$(find teleop)/params/teleop.yaml" command="load"/>
    <node pkg="joy_teleop" type="joy_teleop.py" name="teleop" machine="$(arg machine)" respawn="true" output="screen">
        <param name="robot_name" value="$(env ROBOT)"/>
    </node>
</launch>
